tasks/options/axe-webdriver.js

```diff
@@ -0,0 +1,9 @@
+module.exports = {
+	firefox: {
```

Comment:
Could you please. We use browserstack for cross browser testing on travis. Can axe-webdriver use browserstack? We don't run the performance stuff on browserstack because browserstack is not consistent enough for performance testing.